packages/manager/apps/catalog/src/components/Error/Errors.scss

```diff
@@ -0,0 +1,20 @@
+.manager-error-page {
+  margin-left: auto;
+  margin-right: auto;
+  width: 100%;
+  max-width: 600px;
+  display: grid;
+  height: 100%;
+  overflow: hidden;
+
+  .manager-error-page-image {
+    img {
+      width: 100%;
+    }
+  }
+
+  .manager-error-page-footer {
+    text-align: right;
+    overflow: hidden;
+  }
+}
```

Comment:
We can use tailwind instead of these classes

Reply:
👍